v20.1/admin-ui-jobs-page.md

```diff
@@ -3,31 +3,78 @@ title: Jobs Page
+
+If you need a historical record of all jobs you have run, you should log this information externally. 
+{{site.data.alerts.end}}
+
+## Jobs list
+
+Use the **Jobs** list to see your recently created and completed jobs.
 
-For changefeeds, the table displays a [high-water timestamp that advances as the changefeed progresses](change-data-capture.html#monitor-a-changefeed). This is a guarantee that all changes before or at the timestamp have been emitted. Hover over the high-water timestamp to view the [system time](as-of-system-time.html).
+- For changefeeds, the table displays a [high-water timestamp that advances as the changefeed progresses](change-data-capture.html#monitor-a-changefeed). This is a guarantee that all changes before or at the timestamp have been emitted. Hover over the high-water timestamp to view the [system time](as-of-system-time.html).
 
-The automatic table statistics jobs are not displayed even when the **TYPE** drop-down is set to **All**. To view the automatic statistics creation jobs, filter the results to **Automatic-Statistics Creation** as described in the [Filtering results](#filtering-results) section.
+- [Automatic table statistics](cost-based-optimizer.html#table-statistics) jobs are not displayed even when the **Type** menu is set to **All**. To view these jobs, set **Type** to **Automatic-Statistics Creation** as described [above](#filter-jobs).
 
-## Filtering results
+- To view [job details](#job-details), click on the job description.
 
-You can filter the results based on the status of the jobs or the type of jobs (backups, restores, schema changes, changefeeds, user-created table statistics, and automatic table statistics). You can also choose to view either the latest 50 jobs or all the jobs across all nodes.
+<img src="{{ 'images/v20.1/admin_ui_jobs_page_new.png' | relative_url }}" alt="CockroachDB Admin UI Jobs Page" style="border:1px solid #eee;max-width:100%" />
 
-Filter By | Description
+Parameter | Description
 ----------|------------
-Job Status | From the **Status** menu, select the required status filter.
-Job Type | From the **Type** menu, select **Backups**, **Restores**, **Imports**, **Schema Changes**, **Changefeed**, **Statistics Creation**, or **Auto-Statistics Creation**.
-Jobs Shown | From the **Show** menu, select **First 50** or **All**.
+Description | SQL statement that created the job.
+Job ID | Unique job ID. This value is used to [pause](pause-job.html), [resume](resume-job.html), or [cancel](cancel-job.html) jobs.
+Users | User that created the job.
+Creation Time | Date and time the job was created.
+Status | Current [job status](#job-status) or completion progress. Status can be `PENDING`, `PAUSED`, `FAILED`, `SUCCEEDED`, or `CANCELED`.
+
+### Job status
+
+Status | Description
+-------|------------
+`PENDING` | Job is created but still pending.
```

Comment:
Would be good to use a word other than "pending" in the description.  Maybe something like "hasn't started running yet"? (if that is indeed accurate)